app/assets/stylesheets/ama_layout/layout_components/sections.scss

```diff
@@ -58,6 +62,18 @@
   @include grid-column-end;
 }
 
+.section-centered{
+  &--half{
+    @extend .section;
+    @include large-6;
+    @include large-centered;
+
+    &:last-child:not(:first-child){
```

Comment:
I think instead of this you can just @include grid-column-end; under @include large-centered;